dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/LinkageCheckerMain.java

```diff
@@ -150,16 +187,14 @@ public static void main(String[] arguments)
               .collect(toImmutableSet());
     }
 
-    // TODO this should be a separate method; not part of findLinkageProblems
-    Path outputExclusionFile = linkageCheckerArguments.getOutputExclusionFile();
-    if (outputExclusionFile != null) {
-      ExclusionFiles.write(outputExclusionFile, linkageProblems);
-      System.out.println("Wrote the linkage errors as exclusion file: " + outputExclusionFile);
-      
-      // TODO why do we return an empty set here?
-      return ImmutableSet.of();
-    }
-
     return linkageProblems;
   }
+  
+  private static void writeExclusionFile(Path outputExclusionFile,
```

Comment:
It's unused. I see `Problem.writeExclusionFile` takes care of this.

Reply:
removed